terraform/templates/windows_worker_user_data.tpl

```diff
@@ -20,7 +20,7 @@
 
   # Set Administrator password
   $admin = [adsi]("WinNT://./administrator, user")
-  $admin.psbase.invoke("SetPassword", "${password}")
+  $admin.psbase.invoke("SetPassword", "$password")
```

Comment:
not sure if removing the brackets here is right. I don't see `$password` being set in this script so I assume it must be coming from terraform.

Reply:
@mwrock Ahh, good catch; will fix.

Reply:
Fixed in https://github.com/habitat-sh/builder/pull/1252/commits/6e3ab49d382c9b2cbf5bae52fd5e028fd94a00ce